docs/src/main/paradox/scala/http/routing-dsl/directives/index.md

```diff
@@ -163,6 +163,10 @@ thing like this:
 
 @@snip [DirectiveExamplesSpec.scala](../../../../../../test/scala/docs/http/scaladsl/server/DirectiveExamplesSpec.scala) { #example-2 }
 
+Or better yet using `concat` combinator where we pass each directive as a parameter to the combinator function instead of chaining them with `~` . Let's take a look at the usage of this combinator:
```

Comment:
It can get it's own sub section so that it's easy to link to though - please add a subsection for it :)